common/thorhelper/roxiehelper.hpp

```diff
@@ -65,6 +66,13 @@ class THORHELPER_API HttpHelper : public CInterface
     HttpHelper(StringArray *_validTargets) : validTargets(_validTargets), method(HttpMethod::NONE) {parameters.setown(createProperties(true));}
     inline bool isHttp() { return method!=HttpMethod::NONE; }
     inline bool isHttpGet(){ return method==HttpMethod::GET; }
+    inline bool allowKeepAlive()
+    {
+        const char *connection = queryRequestHeader("Connection");
+        if (!connection)
+            return streq(version, "1.1");
```

Comment:
Would !streq(version, "1.0") be better, in which case the code doesn't have to be changed for newer http versions.